src/net/connection/timeout_thread.rs

```diff
@@ -2,6 +2,7 @@ use super::ConnectionPool;
 use error::{NetworkError, NetworkErrorKind, NetworkResult};
 use events::Event;
 
+use std::sync::atomic::*;
```

Comment:
Could you please specify the types you are using here instead of `*` since we do that in the whole crate.

Reply:
Ah, good catch! I did that as part of my initial workaround, let me clean that up :)